wasmcloud_host/README.md

```diff
@@ -1,21 +1,146 @@
 # wasmCloud Host - Web UI Dashboard
+
 This is the web UI dashboard that provides for a basic way to interact with a host and its associated lattice. This web application automatically starts the [host_core](../host_core/README.md) application as a dependency.
 
-## Starting the Host and Web UI Dashboard
+To learn more about wasmCloud, please view the [Documentation](https://wasmcloud.dev).
+
+
+## Building the host
+
+Clone the repo `wasmCloud/wasmcloud-otp` into some suitable development directory
+
+```console
+git clone https://github.com/wasmCloud/wasmcloud-otp.git
+```
+
+
+### Language Prerequisites
+
+Install the latest version of [Rust](https://www.rust-lang.org/tools/install) and [Elixir](https://elixir-lang.org/install.html).
+Elixir requires the Erlang VM to be installed, but many of the installation methods listed will take care of this for you.
+
+If building the `httpserver` provider from source, you'll also need [Go](https://golang.org/doc/install)
+
+### Environment Prerequisites
+
+1. Since the build of `wasmcloud-otp` will fail on version `16.xx.yy` of NodeJs, you should either install, or switch to, version `14.xx.yy` of NodeJS.
```

Comment:
I wasn't aware of this (I'm using 14), but we should confirm it's still true before submitting the PR